examples/spawn-and-move/src/actions.cairo

```diff
@@ -83,8 +82,7 @@ mod actions {
             );
         }
 
-        fn move(self: @ContractState, direction: Direction) {
-            let world = self.world_dispatcher.read();
+        fn move(direction: Direction, world: IWorldDispatcher) {
```

Comment:
Seeing this, we must inline and remove the `world: IWorldDispatcher` from arguments **if and only if it's the first argument**. We shouldn't allow two arguments with this type either (an other faulty one to add! hehe).

Because at ABI level, removing `world` at first position can be understood. If it's randomly placed in the list of inputs of the function, it can quickly become difficult to call the systems.

Reply:
Thanks ! I will update the code accordingly 👍